samples/backend/python/quart/model/model_utils.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import fields, is_dataclass
+from typing import Type, TypeVar, Union, get_args, get_origin, get_type_hints
+from enum import Enum
+
+T = TypeVar("T")
+
+
+def rename(original_name: str, target_name: str):
```

Comment:
It looks like Pydantic has a standard way of doing this as well-
https://stackoverflow.com/questions/67995510/how-to-inflect-from-snake-case-to-camel-case-post-the-pydantic-schema-validation